### PR TITLE
Add down-arrow on user homepage.

### LINKED ISSUE #3664: on a 13-inch screen the featured project bit of the Zooniverse Home isn't visible
In case the feedback is helpful, I pretty much missed there was a message about stargazing live on the logged in profile page. The only thing that pops up is featured project right at the end of the page on my 13-inch screen. 

screen shot 2017-03-28 at 11 42 42 am

### PR DESCRIPTION
**Pull request by jelliotartz**
Fixes #3664.

Describe your changes.
- Adds a FontAwesome down arrow to user homepage, at bottom of content section, to indicate the presence of featured projects below the content section (13 inch displays only show a small portion of the featured project section).
- I modified the margin on both `home-page-for-user__menu` and `home-page-for-user__down-arrow`, to get the look of an arrow vertically centered between "my collections" and the bottom of the teal box. I played around with other margin adjustments, eg no bottom-margin on `__menu` and identical top- and bottom-margins on `__down-arrow`, and went with what's currently on this branch as what I thought looked best - but happy to change if anyone prefers otherwise.
- Staged [here](https://scroll-down-arrow.pfe-preview.zooniverse.org/).

# Review Checklist

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Did you deploy a staging branch?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [x] Are the tests passing locally and on Travis?

## Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by beckyrother** [comment]
Please reduce the size to 1em and add more space at the top, so the margin-top is 30.

zooniverse

Other than that looks good!

File: app/pages/home-for-user/index.jsx [hunk 1]
```diff
@@ -281,8 +281,12 @@ export default class HomePageForUser extends React.Component {
                 onClose={this.deselectSection}
               />
             )}
-          </div>
 
+            <div className="home-page-for-user__down-arrow">
+              <i className="fa fa-arrow-down fa-2x" aria-hidden="true" />
+            </div>
+
+          </div>
```
Comment (wgranger) [inline app/pages/home-for-user/index.jsx]: It seems a bit unnecessary here, code-wise, to have a `div` with one item in it when you could have the icon standalone. However, this isn't blocking.

**Comment by wgranger** [comment]
screen shot 2017-09-21 at 11 06 03 am

When a section is open the arrow looks a bit odd. It might be good to have the arrow disappear when one of the menu items is opened.

**Comment by beckyrother** [comment]
Good point @wgranger , let's do that.

**Comment by jelliotartz** [comment]
Staging branch updated. Thanks for the suggestions! Agreed, it definitely looks weird having that arrow beneath the open menu items.

**Comment by wgranger** [comment]
LGTM

**Pull request merged**